gcs/src/components/dashboardDataModal.jsx

```diff
@@ -0,0 +1,75 @@
+/*
+    The Modal component for connecting to the Com Port
+    Accessed through the connect button on the navbar
+*/
+
+// Third party imports
+import {
+    Button,
+    Checkbox,
+    Group,
+    LoadingOverlay,
+    Modal,
+    Select,
+    Tooltip,
+  } from '@mantine/core'
+  import { IconInfoCircle, IconRefresh } from '@tabler/icons-react'
+  
+  // Styling imports
+  import resolveConfig from 'tailwindcss/resolveConfig'
+  import tailwindConfig from '../../tailwind.config.js'
+  
+  const tailwindColors = resolveConfig(tailwindConfig).theme.colors
+  const items = ['Item 1', 'Item 2', 'Item 3', 'Item 4', 'Item 5', 'Item 6', 'Item 7'];
+  
+  export default function DashboardDataModal({
+    opened,
+    close,
+    possibleData,
+  }) {
+    return (
+      <Modal
+        opened={opened}
+        onClose={() => {
+          close()
+        }}
+        title='Select Data'
+        centered
+        overlayProps={{
+          backgroundOpacity: 0.55,
+          blur: 3,
+        }}
+        withCloseButton={false}
+      >
+        {/* <LoadingOverlay visible={fetchingComPorts} /> */}
```

Comment:
Is this meant to be commented out?

Reply:
Oh, it's an oversight. I'll deal with that